worlds/peaks_of_yore/__init__.py

```diff
@@ -0,0 +1,172 @@
+        self.multiworld.regions.append(menu_region)
+
+        cabin_region = Region("Cabin", self.player, self.multiworld)
+        self.multiworld.regions.append(cabin_region)
+        menu_region.connect(cabin_region)
+
+        if self.options.enable_fundamental:
+            fundamentals_region = Region("Fundamentals", self.player, self.multiworld)
+
+            self.peaks_in_pool.extend(get_location_names_by_type(0, "Peak"))
+            self.artefacts_in_pool.extend(get_location_names_by_type(0, "Artefact"))
+
+            fundamentals_region.add_locations(get_locations(0), PeaksOfYoreLocation)
+            cabin_region.connect(fundamentals_region, "Fundamentals Book",
+                                 lambda state: state.has("Fundamentals Book", self.player))
+            self.location_count += len(fundamentals_region.locations)
+
+        if self.options.enable_intermediate:
+            intermediate_region = Region("Intermediate", self.player, self.multiworld)
+
+            self.peaks_in_pool.extend(get_location_names_by_type(1, "Peak"))
+            self.artefacts_in_pool.extend(get_location_names_by_type(1, "Artefact"))
+
+            intermediate_region.add_locations(get_locations(1), PeaksOfYoreLocation)
+            cabin_region.connect(intermediate_region, "Intermediate Book",
+                                 lambda state: state.has("Intermediate Book", self.player))
+            self.location_count += len(intermediate_region.locations)
+
+        if self.options.enable_advanced:
+            advanced_region = Region("Advanced", self.player, self.multiworld)
+
+            self.peaks_in_pool.extend(get_location_names_by_type(2, "Peak"))
+            self.artefacts_in_pool.extend(get_location_names_by_type(2, "Artefact"))
+
+            advanced_region.add_locations(get_locations(2), PeaksOfYoreLocation)
+            cabin_region.connect(advanced_region, "Advanced Book",
+                                 lambda state: state.has("Advanced Book", self.player))
+            self.location_count += len(advanced_region.locations)
+
+        if self.options.enable_expert:
+            expert_region = Region("Expert", self.player, self.multiworld)
+            expert_locations: dict[str, int] = {k: v for k, v in get_locations(3).items() if
+                                                (not self.options.disable_solemn_tempest) or v != 37}
+
+            self.artefacts_in_pool.extend(get_location_names_by_type(3, "Artefact"))
+            self.peaks_in_pool.append("Great Bulwark")
+            if not self.options.disable_solemn_tempest:
+                self.peaks_in_pool.append("Solemn Tempest")
+
+            expert_region.add_locations(expert_locations, PeaksOfYoreLocation)
+            cabin_region.connect(expert_region, "Expert Book", lambda state: state.has("Expert Book", self.player))
+            self.location_count += len(expert_region.locations)
+
+    def create_items(self) -> None:
```

Comment:
Please add a `def get_filler_item_name(self) -> str:` somewhere. It should return the name of a filler item that you are okay with your world creating extra copies of. It's usually called during something like plando, item links (I think), or the start inventory panic method in the host.yaml.

You can probably just rework `create_extra_item` into it, or to use it directly